server/src/controllers/Events.js

```diff
@@ -66,22 +66,24 @@ module.exports = {
         };
       }
       const event = await Event.findByIdAndUpdate(eventId, req.body);
-      res.json({
+      return res.status(200).json({
         id: event._id
       });
     } catch (err) {
-      res.status(400).send({
-        err: err
-      });
+      return res.status(400).json({
+        message: err.message,
+      })
     }
   },
+
   async deleteEvent(req, res) {
     const eventId = req.params.id;
     const event = await Event.findById(eventId);
     await event.remove();
     await cloudinary.v2.uploader.destroy(eventId);
-    res.status(204);
+    return res.status(204).json({message:"Successfully deleted event"});
   },
```

Comment:
A status of 204 indicates No Content (see [here](https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/204)). It'd be better to remove the message and pass in an empty object here.

Reply:
Alright.

Reply:
Should we pass an empty object, or nothing at all?  
Lets keep it uniform. We have passed an empty obj somewhere and nothing in some places